openpype/hosts/blender/utility_scripts/make_paths_relative.py

```diff
@@ -15,17 +16,12 @@
     )
     errors = []
     # Resolve path from source filepath with the relative filepath
-    for datablock in chain(bpy.data.libraries, bpy.data.images):
+    for datablock in get_datablocks_with_filepath(relative=False):
         try:
-            if (
-                datablock
-                and not datablock.is_library_indirect
-                and not datablock.filepath.startswith("//")
-            ):
-                datablock.filepath = bpy.path.relpath(
-                    str(Path(datablock.filepath).resolve()),
-                    start=str(Path(bpy.data.filepath).parent.resolve()),
-                )
+            datablock.filepath = bpy.path.relpath(
+                str(Path(datablock.filepath).resolve()),
+                start=str(Path(bpy.data.filepath).parent.resolve()),
```

Comment:
Does this still work on windows when the datablock is on one drive, and the workfile on another?

Reply:
Let's say you got your `work` site on your main drive and your `main` site on another drive.
On linux there's no issue having a relative path that way whatever the paths are, but what about windows? If your `work` is on your `C` drive and your `main` on `D` for instance, I'm not sure this works. I've seen errors about the path being on one drive and start on another.

Reply:
Thanks for the clarification